stories/imdb/stories/greyVest.js

```diff
@@ -71,6 +71,7 @@ let tree = Contexture({
           key: 'titleText',
           type: 'tagsText',
           field: 'title',
+          paused: true,
```

Comment:
You should do this to a different node because this is the only `tagsText` one we have in the tree. Pick one that isn't unique like one of the facet or just add another.